### PR TITLE
bitwise: update license

### PR DESCRIPTION
**Pull request by pixelcaliber**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/b/bitwise.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Bitwise < Formula
   homepage "https://github.com/mellowcandle/bitwise"
   url "https://github.com/mellowcandle/bitwise/releases/download/v0.50/bitwise-v0.50.tar.gz"
   sha256 "806271fa5bf31de0600315e8720004a8f529954480e991ca84a9868dc1cae97e"
-  license "GPL-3.0"
+  license "GPL-3.0-only"
```
Comment (SMillerDev) [inline Formula/b/bitwise.rb:6]: Not sure if that is true: https://github.com/mellowcandle/bitwise/blob/master/src/shunting-yard.c#L3-L4

**Comment by Moisan** [comment]
I think we can base ourselves on  https://github.com/mellowcandle/bitwise/blob/0a4e7d93b3e922e2293251fdbfd0e4fb1cc7910b/debian/copyright#L30

File: Formula/b/bitwise.rb (hunk 1 above)
Comment (pixelcaliber) [reply Formula/b/bitwise.rb:6]: "GPL-3.0" to "GPL-3.0-only" in the formula's license specification is due to a change in the SPDX, old identifier "GPL-3.0" was ambiguous. To address this ambiguity, they introduced more specific identifiers 
https://spdx.org/licenses/

**Comment by pixelcaliber** [comment]
@chenrui333 can you please help why commit style check is failing?

**Comment by pixelcaliber** [comment]
@SMillerDev hey could you please help, why commit style checks is getting failed? this is my first time contribution. Thanks!

**Comment by bevanjkay** [comment]
@pixelcaliber To pass the `Commit Style` check the commits need to be squashed. Otherwise once approved, a replacement PR will be dispatched (with the commit still attributed to you).

**Review by p-linnane — APPROVED** [review]

**Pull request merged**